chrome/model.js

```diff
@@ -223,6 +219,79 @@ GmailToTrello.Model.prototype.loadTrelloMembers = function(boardId) {
     });
 };
 
+GmailToTrello.Model.prototype.loadThreadTrelloCards = function () {
+    var self = this;
+
+    self.isThreadCardsLoaded = true;
+
+    // get cards associated with the thread on the current thread
+    // i would love to do this with the message ids, but there's no way to get all of them to search
+    // so let's use the subject and time method
+    if (!self.gmail || !self.gmail.subject || self.gmail.subject.length < 1) {
+        gtt_log('loadThreadTrelloCards: Current page is not a thread');
+        self.trello.threadCards = [];
+        self.event.fire('onLoadThreadTrelloCardsSuccess');
+    }
```

Comment:
If there's no subject line on the current page, then it's a landing page, so I assume it doesn't have any associated Trello cards and we just skip this request.